diff --git a/h5peditor-file.class.php b/h5peditor-file.class.php
index 89329e1..ebda1a6 100644
--- a/h5peditor-file.class.php
+++ b/h5peditor-file.class.php
@@ -5,7 +5,7 @@
  */
 class H5peditorFile {
   private $result, $field, $interface;
-  public $type, $name, $path, $mime, $size;
+  public $type, $name, $path, $mime, $size, $extension;
 
   /**
    * Constructor. Process data for file uploaded through the editor.
diff --git a/h5peditor.class.php b/h5peditor.class.php
index 5bbcdfd..10cc4b6 100644
--- a/h5peditor.class.php
+++ b/h5peditor.class.php
@@ -51,7 +51,7 @@ class H5peditor {
     'ckeditor/ckeditor.js',
   );
   private $h5p, $storage;
-  public $ajax, $ajaxInterface;
+  public $ajax, $ajaxInterface, $content;
 
   /**
    * Constructor for the core editor library.
@@ -630,7 +630,7 @@ class H5peditor {
           'majorVersion' => $local_lib->major_version,
           'minorVersion' => $local_lib->minor_version,
           'patchVersion' => $local_lib->patch_version,
-          'patchVersionInFolderName' => $local_lib->patch_version_in_folder_name
+          'patchVersionInFolderName' => $local_lib->patch_version_in_folder_name ?? 0
         ]);
         $icon_path = $this->h5p->h5pF->getLibraryFileUrl($library_folder, 'icon.svg');
       }
